src/mumble/Messages.cpp

```diff
@@ -45,8 +45,11 @@
 	}
 
 #define SELF_INIT \
-	ClientUser *pSelf = ClientUser::get(g.uiSession);
-
+	ClientUser *pSelf = ClientUser::get(g.uiSession); \
+	if (! pSelf) { \
```

Comment:
```suggestion
	if (!pSelf) { \
```

We actually have multiple instances of `if (! condition)` across the codebase .

Reply:
I copied that style from 5 lines above or so. When in Rome....

Reply:
No worries, we will eventually correct all instances.